code/modules/mob/living/silicon/pai/pai_shell.dm

```diff
@@ -32,8 +32,6 @@
 		client.perspective = EYE_PERSPECTIVE
 		client.eye = src
 	var/turf/T = get_turf(card.loc)
-	card.loc = T
```

Comment:
this one here seems to move the card specifically to the turf, then to the src, so you might want
card.forceMove(T)
card.forceMove(src)

as it may have some reason to do that